### ISSUE TITLE
Output defaults for OpenFAST

### ISSUE THREAD

**Issue by mayankchetan** [deslaughter/acdc-app#5]
Current defaults in the output requested from OpenFAST results in very large files when running BeamDyn. Also, both `.out` and `.outb` are being written, this also uses larger memory (and swap) footprint when using BermDyn time steps.

Recommended parameters:

 - `SttsTime` = DT*10
 - `DT_Out` = 0.01
 - `OutFileFmt` = 2

**Comment by deslaughter** [comment]
I think we should go with `OutFileFmt` = 1 so OpenFAST doesn't preallocate a ton of memory. I feel like `SttsTime` = DT*10 will give a huge number of outputs, but I can give it a shot.

**Comment by mayankchetan** [comment]
`OutFileFmt = 1` was what I was thinking but typed out `2`. For the status, whatever makes the best sense for rest of your code, since the progress bar is a SWEET feature!

**Comment by deslaughter** [comment]
As discussed in #4, setting `SttsTime` to a value less than 1 second generates the same status message multiple times without which won't update the progress bar. I'm going to leave `DT_Out` unmodified from what was originally imported from the input file until we have a compelling reason to make it editable inside ACDC. I've updated the code to set `OutFileFmt` to `1` as we discussed. These changes have been implemented in release v0.3.0-alpha.
